### PR TITLE
Add radar charts (#305)

### PR DESCRIPTION
**Pull request by haotran-avaiga**
Adresses #305.

### CODE REVIEW

File: docs/manuals/gui/viselements/charts/radar.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+<figure>
+    <img src="radar-multi-trace-dark.png" class="visible-dark" />
+    <img src="radar-multi-trace.png" class="visible-light" />
+    <figcaption>Multiple Trace Radar Chart</figcaption>
+</figure>
+
+Explanation of parameters and options relevant to radar charts can be found [here](https://plotly.com/javascript/reference/scatterpolar/). Full details for the Taipy Chart control can be found in the Chart Control section of the [Taipy GUI User Manual](https://docs.taipy.io/en/latest/manuals/gui/viselements/chart/)
+
+### Key parameters
```
Comment (dinhlongviolin1) [inline docs/manuals/gui/viselements/charts/radar.md]: Should be placed at the start of file after description of chart types

**Review by dinhlongviolin1 — APPROVED** [review]
Working demos, good examples

File: docs/manuals/gui/viselements/charts/radar.md (hunk 1 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/charts/radar.md]: Done.

**Review by FredLL-Avaiga — APPROVED** [review]

**Review by dinhlongviolin1 — APPROVED** [review]
Not much to say

**Pull request merged**